kpi/tasks.py

```diff
@@ -26,4 +28,14 @@ def sync_kobocat_xforms(
         username=username,
         quiet=quiet,
         populate_xform_kpi_asset_uid=populate_xform_kpi_asset_uid,
-    )
\ No newline at end of file
+    )
+
+    
+@shared_task
+def sync_media_files(asset_uid):
+    from kpi.models.asset import Asset  # avoid circular imports
+
+    asset = Asset.objects.get(uid=asset_uid)
+    asset.deployment.sync_media_files()
+    # If no exceptions have been raised, let's tag the deployment has synced
+    asset.deployment.set_status(asset.deployment.STATUS_SYNCED)
```

Comment:
what happens if a couple of these get going simultaneously on the same asset? anything bad?

Reply:
not really: the last one to run should overwrite the results from previous ones